public/static/locales/cy/pageTitles.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
No need for these separate translation files anymore.  All translations need to be stored in single 'common' file to prevent loading delay on front end